nutaudet/NuTauMudet.cxx

```diff
@@ -511,8 +508,7 @@ void NuTauMudet::ConstructGeometry()
       TGeoVolume *volIron = new TGeoVolume("volIron",IronLayer,Iron);
       volIron->SetLineColor(kGray);
 
-      TGeoBBox *IronLayer1 = new TGeoBBox("Iron",fXFe/2, fYFe/2, fZFethin/2);
-      TGeoVolume *volIron1 = new TGeoVolume("volIron1",IronLayer1,Iron);
+      TGeoBBox *IronLayer1 = new TGeoBBox("Iron",fXFe/2, fYFe/2, fZFethin/2);      
```

Comment:
No need to repeat the class name, `auto*` would make this less verbose

Reply:
Ok! For now I have just done to this line, but I guess it would apply to all of geometry code lines in the future.